src/app/app.component.ts

```diff
@@ -1,51 +1,49 @@
-import { Component, ViewEncapsulation,OnInit } from '@angular/core';
+import { Component, ViewEncapsulation, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 import { NgbModal, NgbModalConfig } from '@ng-bootstrap/ng-bootstrap';
 import { AuthService } from './auth/auth.service';
+
 @Component({
   selector: 'app-root',
   encapsulation: ViewEncapsulation.None,
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
-export class AppComponent implements OnInit {
+export class AppComponent {
   modal: any;
   profileModal: any;
-  isMenuActive: boolean = false;
-  isActive: boolean = false;
-  user;
-
+  isMenuActive = false;
+  isActive = false;
   constructor(
     public authService: AuthService,
     config: NgbModalConfig,
     private modalService: NgbModal,
     private profileModalService: NgbModal,
+    public router: Router
   ) {
     config.backdrop = 'static';
   }
-  open(content) {
+  open(content: any): void {
     this.modal = this.modalService.open(content, {
       animation: true,
       windowClass: 'form-modal',
     });
   }
-  openProfile(content) {
+
+  openProfile(content: any): void {
     this.profileModal = this.profileModalService.open(content, {
       animation: true,
       windowClass: 'profile-modal',
```

Comment:
We can move the `{ backdrop: static}` thing here if it is for the same ngbModal?

Reply:
yuss